.gitignore

```diff
@@ -1,6 +1,7 @@
 # dependencies
 node_modules/
 packages/*/package-lock.json
+packages/**/test/**/yarn.lock
```

Comment:
Wdyt of adding a separate .gitigore in `packages/nextjs/test/types` instead of adding this here? This has a lot of wildcards and might lead to headaches in the future where we're wondering why a lockfile isn't tracked.